### PR TITLE
feat: ability to supply openjd-sessions Python wheel

### PR DESCRIPTION
**Pull request by jusiskin**
### What was the problem/requirement? (What/Why)

There is no ability to supply a custom developer version of `openjd-sessions` and have it deployed with the worker instances launched by `deadline-cloud-test-fixtures`

### What was the solution? (How)

Introduce a new `OPENJD_SESSIONS_WHL_PATH` environment variable that contains a glob pattern resolving to a local path to a `.whl` file for `openjd-sessions` on the test machine. This works the same as the existing `WORKER_AGENT_WHL_PATH` environment variable for supplying a custom developer version of the worker agent.

### What is the impact of this change?

Consumers of the library can orchestrate tests of workers using a custom supplied `openjd-sessions` Python library.

### How was this change tested?

1.  Modified the worker agent e2e test suite to extend the `worker_config` fixture from `deadline-cloud-test-fixtures` instead of reimplementing it
2. Set the `OPENJD_SESSIONS_WHL_PATH` to the path of a developer-built `.whl` file
3. Ran the tests
4. Inspected the worker logs sent to CloudWatch and confirmed the `openjd-sessions` version reported at the top matched the local version specified in `OPENJD_SESSIONS_WHL_PATH`

### Was this change documented?

Yes, the new environment variable was added to `src/deadline_test_fixtures/example_config.sh` where the other environment variables are documented.

### Is this a breaking change?

No

----

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: src/deadline_test_fixtures/fixtures.py [hunk 1]
```diff
@@ -453,14 +456,39 @@ def worker_config(
...
+    openjd_sessions_whl_path = os.getenv("OPENJD_SESSIONS_WHL_PATH")
+    if openjd_sessions_whl_path:
+        LOG.info(f"Using openjd-sessions agent whl file: {openjd_sessions_whl_path}")
+        resolved_whl_paths = glob.glob(openjd_sessions_whl_path)
+        assert (
+            len(resolved_whl_paths) == 1
+        ), f"Expected exactly one openjd-sessions whl path, but got {resolved_whl_paths} (from pattern {openjd_sessions_whl_path})"
+        resolved_whl_path = resolved_whl_paths[0]
+
+        if operating_system.name == "AL2023":
```
Comment (AWS-Samuel) [inline src/deadline_test_fixtures/fixtures.py]:
```suggestion
        if operating_system.name.startswith("AL"):
```

Optional, but if we ever end up needing to support a new edition of AL then this becomes one less place to modify

**Review by AWS-Samuel — DISMISSED** [review]

File: src/deadline_test_fixtures/fixtures.py (hunk 1 above)
Comment (jusiskin) [reply src/deadline_test_fixtures/fixtures.py]: I went ahead and added this along with `OperatingSystem.is_amazon_linux()` and `OperatingSystem.is_windows()` methods to abstract these checks.

Comment (AWS-Samuel) [reply src/deadline_test_fixtures/fixtures.py]: Nice, even better. Thanks!

**Review by AWS-Samuel — APPROVED** [review]

File: src/deadline_test_fixtures/fixtures.py [hunk 2]
```diff
@@ -444,23 +447,52 @@ def worker_config(
...
+                "C:\\Windows\\System32\\Config\\systemprofile\\AppData\\Local\\Temp",
+                os.path.basename(resolved_whl_path),
+            )
+        else:
+            raise NotImplementedError(f"Unsupported operating system: {operating_system.name}")
+        file_mappings.append((resolved_whl_path, dest_path))
+        LOG.info(
+            f"The openjd-sessions whl file will be copied to {dest_path} on the Worker environment"
+        )
+        pip_install_requirement_specifiers.append(dest_path)
```
Comment (godobyte) [inline src/deadline_test_fixtures/fixtures.py:495]: nit - this logic looks very similar to the worker agent whl install, can there be a helper to reduce duplicated code?

**Review by godobyte — APPROVED** [review]

**Pull request merged**